modules/button/react/lib/ButtonStyles.ts

```diff
@@ -335,6 +335,11 @@ export const iconButtonStyles: ButtonGenericStyle = {
         height: canvas.spacing.l,
         ...getIconButtonStateStyle(IconButtonVariant.Square),
       },
+      [IconButtonVariant.Alternate]: {
+        borderRadius: borderRadius.m,
+        width: '32px !important', // override width auto for small buttons
```

Comment:
any reason why we have width auto on icon button size small? All of our icons are always 24 by 24